dotnet/src/SemanticKernel.Abstractions/Filters/Prompt/PromptRenderedContext.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Class with data related to prompt after rendering.
+/// </summary>
+[Experimental("SKEXP0005")]
+public sealed class PromptRenderedContext : PromptFilterContext
+{
+    private string? _renderedPrompt;
```

Comment:
Why is this nullable?

Reply:
@stephentoub Constructor still shows a warning that `_renderedPrompt` is not initialized:
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/55f63cbf-5663-4507-83e4-ea52202ade17)

Reply:
You need to say what member isn't null in the attribute. I'm surprised that even compiles without it.

Reply:
Fixed by placing attribute on property setter.